stats/view/worker.go

```diff
@@ -61,42 +61,43 @@ func Find(name string) (v *View) {
 	return resp.v
 }
 
-// Deprecated: Registering is a no-op. Use the Subscribe function.
-func Register(_ *View) error {
-	return nil
+// Deprecated: Use the Register function
+func Subscribe(v ...*View) error {
+	return Register(v...)
 }
 
-// Deprecated: Unregistering is a no-op, see: Unsubscribe.
-func Unregister(_ *View) error {
+// Deprecated: Use the Unregister function
+func Unsubscribe(v ...*View) error {
+	Unregister(v...)
 	return nil
 }
 
-// Deprecated: Use the Subscribe function.
-func (v *View) Subscribe() error {
-	return Subscribe(v)
+// Deprecated: Use the Register function.
+func (v *View) Register() error {
```

Comment:
This should not be renamed since the method is only kept for backwards compatibility.

Reply:
Gotcha, thanks. It is a vestige of my earlier commits that I thought I had reverted. I'll fix it right now.